Insert attachments documentation (#163)
exoplatform/exo-documentation: docs/GettingStarted.rst

|image196|

**Insert an attachment**

> You can insert attachments in the article's content by clicking in the appopriate icon of the creation form. 

Reviewer: ```suggestion
You can insert attachments in the article's body by clicking on the paper clip icon in the creation form. 
```

|image199|

> In the displayed drawer, you can either upload manually your files or using a drag and drop in the appropriate area. You can add up to 10 files and each file must not exceed 25 Mb.

Reviewer: ```suggestion
A drawer panel opens where you can either upload by selecting your files or by using a drag and drop in the dedicated area. You can attach up to 10 files and each file must not exceed 25 MB.
```

|image200|

> Once you add files, you can see the upload progress in the drawer as well as the number of selected files. You can cancel the upload of a file in progress or delete it if it is already uploaded using the close icon.

Reviewer: ```suggestion
Once you add files, you can see the upload progress as well as the number of selected files. You can cancel the upload of a file in progress or delete it if it is already uploaded using the close icon.
```

|image201|

> You can close the drawer using the 'X' icon, the close button or simply by clicking outside the drawer area. If the upload is still in progress, a circular movement will be visible on the icon containing the number of attachments and the post button will be disabled until all files are uploaded.

Reviewer: ```suggestion
You can close the drawer using the 'X' icon, the close button or simply by clicking outside the drawer panel. If the upload is still in progress as you close the panel, it will continue in the background. An animation around the attachments counter badge indicates progress. Note that the post button will be disabled until all files are uploaded.
```

|image161|

The attachments list will be available in the articles details after posting it.  

|image203|

> You can display the file preview by clicking on it.

Reviewer: ```suggestion
You can preview the file by clicking on it.
```
